target/linux/ath79/dts/ar9132_buffalo_wzr-hp-g300nh.dts

```diff
@@ -0,0 +1,259 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9132.dtsi"
+
+/ {
+	compatible = "buffalo,wzr-hp-g300nh", "qca,ar9132";
+	model = "Buffalo WZR-HP-G300NH";
+
+	aliases {
+		led-boot = &led_diag;
+		led-failsafe = &led_security;
+		led-upgrade = &led_diag;
+	};
+
+	clock40mhz: ref {
+		compatible = "fixed-clock";
+		#clock-cells = <0>;
+		clock-frequency = <40000000>;
+	};
+
+	gpio2: gpio-extender {
+		compatible = "nxp,74hc153-gpio";
+		gpio-controller;
+		#gpio-cells = <2>;
+
+		// GPIOs used by this node
+		gpio-s0 = <&gpio 9 GPIO_ACTIVE_HIGH>;
+		gpio-s1 = <&gpio 11 GPIO_ACTIVE_HIGH>;
+		gpio-1y = <&gpio 12 GPIO_ACTIVE_HIGH>;
+		gpio-2y = <&gpio 14 GPIO_ACTIVE_HIGH>;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
```

Comment:
Please try with "gpio-keys", poll-interval can be dropped.

Reply:
I would be happy to use gpio-keys but I get this in the log:

    [ 3.498802] gpio-keys keys: failed to get irq for gpio:504
    [ 3.504409] gpio-keys keys: failed to get irq for gpio:505
    [ 3.509944] gpio-keys keys: failed to get irq for gpio:506
    [ 3.515479] gpio-keys keys: failed to get irq for gpio:507
    [ 3.521017] gpio-keys keys: failed to get irq for gpio:509
    [ 3.526554] gpio-keys keys: failed to get irq for gpio:510
    [ 3.532091] gpio-keys keys: failed to get irq for gpio:511